src/utils/hooks/useDeeplinks.ts

```diff
@@ -96,6 +98,7 @@ export const useUrlEventHandler = () => {
 export const useShopDeepLinkHandler = () => {
   const navigation = useNavigation();
   const availableGiftCards = useAppSelector(selectAvailableGiftCards);
+  const categories = useAppSelector(selectCategoriesWithIntegrations);
   const integrations = useAppSelector(selectIntegrations);
 
   const shopDeepLinkHandler = (
```

Comment:
For some reason, this function is never called.

Reply:
Thanks for catching this. Should work now.